sources/packages/web/src/views/institution/EditInstitutionLocation.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <full-page-container :noSidebar="true">
```

Comment:
`noSidebar` is some leftover property that no longer exists?

Reply:
ya, it was a left over from the last PR. so i removed it in this PR